docs/claims.md

```diff
@@ -63,16 +63,16 @@ maximum security), you can start by navigating to the Claims app on Polkadot-JS
 account you would like to claim the DOT into and click the blue "Continue" button to proceed. Your
```

Comment:
Can you also include https://wiki.polkadot.network/docs/en/maintain-guides-democracy as an updated page?